git/services/versionInfo.js

```diff
@@ -36,15 +38,15 @@ function getCodeName(tagName) {
  * @return {String} The commit SHA
  */
 function getCommitSHA() {
-  return child.spawnSync('git', ['rev-parse', 'HEAD'], {encoding:'utf8'}).stdout.replace('\n', '');
+  return child.spawnSync(GIT, ['rev-parse', 'HEAD'], {encoding:'utf8'}).stdout.replace('\n', '');
 }
 
 /**
  * Compute a build segment for the version, from the Jenkins build number and current commit SHA
  * @return {String} The build segment of the version
  */
 function getBuild() {
-  const hash = child.spawnSync('git', ['rev-parse', '--short', 'HEAD'], {encoding:'utf8'}).stdout.replace('\n', '');
+  const hash = child.spawnSync(GIT, ['rev-parse', '--short', 'HEAD'], {encoding:'utf8'}).stdout.replace('\n', '');
```

Comment:
I know this is unrelated to this PR- but it's interesting that there are also git-local commands (i.e. not consulting a remote). These commands will execute from within the execroot and not in the actual Git repository. I'm not sure if this is an issue, but maybe you have some thoughts. I could see:

Ideally we would properly consume the stamped Bazel workspace status information (we use that for the NPM package releasing already). Though that may be more difficult with the npm_package_bin rule)

Reply:
Yeah that's a good point. I'll go through the cases where any local version info is used in the dgeni build and make sure those are versions from the workspace status command.

Reply:
Added that follow-up to the planning doc.